README.md

```diff
@@ -1,9 +1,15 @@
 ![Open Service Broker API](https://www.openservicebrokerapi.org/wp-content/uploads/2016/12/osbapi_logo_concept3_wtm.png)
 
-[The Open Service Broker API specification](spec.md)
+## Latest Release: v2.11
+[Read the specification](https://github.com/openservicebrokerapi/servicebroker/tree/v2.11)
 
 [Release Notes for past versions](release-notes.md)
 
+## Working Changes
```

Comment:
I'm assuming this new section is for changes that have been through the "implementation" phase and are accepted to be part of the next rev of the spec but we just haven't released a new version - correct?  IOW, this isn't for changes that are still under "implementation review".